vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java

```diff
@@ -70,15 +70,17 @@ public LocatorType getType()
                 ? waitForElement(searchContext, locator)
                 : searchContext.findElements(locator);
         boolean elementsFound = null != elements;
-        LOGGER.info("Total number of elements found {} is equal to {}", locator, elementsFound ? elements.size() : 0);
+        LOGGER.atInfo().addArgument(locator)
+                       .addArgument(() -> elementsFound ? elements.size() : 0)
+                       .log("Total number of elements found {} is equal to {}");
```

Comment:
```
"Total number of elements found {} is {}"
```
?

Reply:
what if someone uses this message in known issue?